src/components/ListComposable/utils/scrollToItem.ts

```diff
@@ -0,0 +1,17 @@
+import type {ListItemId} from '../types';
+
+import {createListItemQa} from './createListItemQa';
+
+export const scrollToItem = (itemId: ListItemId, containerRef?: HTMLDivElement) => {
+    if (document) {
+        const element = (containerRef || document).querySelector(
+            `[data-qa="${createListItemQa(itemId)}"]`,
```

Comment:
qa attributes are designed to be used in tests. Is there another way to find element?

Reply:
now it is assumed that there will be built-in support for qa attributes in new list.

The old component cloned the elements and added a ref there, through which the upstream component gained access to DOM api. This is very expensive solution which may contribute to an uncontrolled decrease in performance. We should avoid this

Reply:
Maybe we should use another name for this attribute?